requirements.txt

```diff
@@ -1,6 +1,33 @@
-matplotlib==3.0.2
-PySide==1.2.4
-# PySide2==5.12.0
-Shiboken==1.2.2
-six==1.12.0
-Markdown==3.2.2
+area==1.1.1
+bzr_pqm==1.4.0
```

Comment:
where is this used?

Reply:
[src/Tools/updateppa.py](https://github.com/FreeCAD/FreeCAD/blob/b4780be1d1b0358fef4f0b292924c7d333190c71/src/Tools/updateppa.py) 
Probably can be omitted.